project/Dependencies.scala

```diff
@@ -92,6 +96,7 @@ object Dependencies {
     Compile.akkaHttp2Support,
     Compile.akkaDiscovery,
     Compile.akkaHttpCors % "provided",
+    Compile.commonProtos,
```

Comment:
Is this also needed when you're generating Java code, so not using scalapb? If so then perhaps it should come in via the generators instead of always being part of the runtime?

Reply:
Oh yeah I'll try that!